context/app/static/js/components/detailPage/provenance/ProvGraph/ProvGraph.spec.js

```diff
@@ -17,40 +17,35 @@ document.getElementsByClassName = () => [
   },
 ];
 
-const server = setupServer(
-  rest.post(`/${appProviderEndpoints.elasticsearchEndpoint}`, (req, res, ctx) => {
-    const descendantData = {
-      hits: {
-        hits: [
-          {
-            _source: {
-              immediate_descendants: [
-                {
-                  uuid: sampleDescendantsProv[0].uuid,
-                },
-                {
-                  uuid: sampleDescendantsProv[1].uuid,
-                },
-              ],
+const descendantData = {
+  hits: {
+    hits: [
+      {
+        _source: {
+          immediate_descendants: [
+            {
+              uuid: sampleDescendantsProv[0].uuid,
+            },
+            {
+              uuid: sampleDescendantsProv[1].uuid,
             },
-          },
-        ],
+          ],
+        },
       },
-    };
-    return res(ctx.json(descendantData), ctx.status(200));
+    ],
+  },
+};
+
+const server = setupServer(
+  http.post(`/${appProviderEndpoints.elasticsearchEndpoint}`, () => {
```

Comment:
Why `appProviderEndpoints.elasticsearchEndpoint` and not `appProviderEndpoints.searchEndpoint`? The dependency is to search-api.

Reply:
This property name was already there (my changes in this file were to update the usage of `mock-service-worker`). We still refer to the search API endpoint as the `elasticsearchEndpoint` in the data that is provided by flask:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/c93afdbd-a930-4773-aa34-6724a94f1282)

I've created a ticket to revise this sitewide: https://hms-dbmi.atlassian.net/browse/CAT-710